parsl/dataflow/dflow.py

```diff
@@ -606,7 +606,18 @@ def sanitize_and_wrap(self, task_id, args, kwargs):
         # Replace item in args
         new_args = []
         for dep in args:
-            if isinstance(dep, Future):
+            if isinstance(dep, list) or isinstance(dep, tuple):
+                new_sub_args = []
```

Comment:
does this mean that if I pass in a future of a tuple to an app, that app will see a `list` rather than a `tuple` as its parameter?